sklearn/ensemble/_voting.py

```diff
@@ -91,6 +91,27 @@ def fit(self, X, y, sample_weight=None):
 
         return self
 
+    def fit_transform(self, X, y=None, **fit_params):
+        """Return class labels or probabilities for each estimator.
+
+        Return predictions for X for each estimator.
+
+        Parameters
+        ----------
+        X : {array-like, sparse matrix, dataframe} of shape \
+                (n_samples, n_features)
+            Input samples.
+        y : ndarray of shape (n_samples,), default=None
+            Target values (None for unsupervised transformations).
+        **fit_params : dict
+            Additional fit parameters.
+        Returns
+        -------
+        X_new : ndarray array of shape (n_samples, n_features_new)
+            Transformed array.
+        """
+        super().fit_transform(X, y, **fit_params)
```

Comment:
The following change:

```suggestion
        return super().fit_transform(X, y, **fit_params)
```

fixes the testing failures.

Reply:
hey, @alfaro96 thanks for your help. Finally, all tests are pass.
Your suggestions really helped.